external/corefx-bugfix/src/Common/src/Interop/Windows/kernel32/Interop.CopyFile.Uap.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should this file be `*.Uap.cs` or `*.Uwp.cs`?

Reply:
This is a copy of the existing .Uap.cs file in the corefx submodule, so I left the name the same. I'm unsure what the correct name is though.